### PR TITLE
HW-129 - Adding README.md file and changing info.xml.

### PR DESCRIPTION
**Pull request by coldrunKacper**
Adding README.md and changing info.xml.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Installation
+------
+
+Go to 
+- Administer -> System Settings -> Manage Extensions (for CiviCRM < 4.7)
+- Administer -> System Settings -> Extensions (for CiviCRM >= 4.7)
+
+and install Activity Report (uk.co.compucorp.civicrm.activityreport) extension.
+
+No additional steps is required.
```
Comment (nishant-bhorodia) [inline README.md]: should this be "No additional steps **are** required" ?

**Comment by nishant-bhorodia** [comment]
Approved !

**Comment by jamienovick** [comment]
Can we also mention that activity custom fields are supported but activity
contacts are not at this point.

I think we should refer to the library we are using for peoples reference.

Does it create a menu link? If so we should say that too.

How to use - quick explanation on what you can do with it.

**Comment by jamienovick** [comment]
Oh ignor me. You did it all. Great job!

We should get this published for automatic download on civicm site once
signed off testing

**Pull request merged**